wandb/sdk/launch/utils.py

```diff
@@ -48,7 +48,7 @@
     r"^https?://localhost"
 )  # for testing, not sure if we wanna keep this
 
-API_KEY_REGEX = r"WANDB_API_KEY=\w+"
+API_KEY_REGEX = r"WANDB_API_KEY=\w+(-\w+)?"
```

Comment:
Previously this might allow output of the api key if it had a hyphen (as in local)